source/fe/mapping_fe_field.inst.in

```diff
@@ -19,6 +19,10 @@ for (deal_II_dimension : DIMENSIONS; deal_II_space_dimension : SPACE_DIMENSIONS;
      VEC : REAL_VECTOR_TYPES)
   {
 #if deal_II_dimension <= deal_II_space_dimension
+    template class MappingFEField<deal_II_dimension,
+                                  deal_II_space_dimension,
+                                  VEC>;
+
     template class MappingFEField<
```

Comment:
Do we need to instantiate the old class?

Reply:
Good point! The old class only appears in the header file, so we don't have to explicitly instantiate it.